engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -570,7 +569,10 @@ void paladin_t::target_mitigation( school_e school,
 
   if ( standing_in_consecration() )
   {
-    s -> result_amount *= 1.0 + ( cache.mastery() * mastery.divine_bulwark -> effectN( 2 ).mastery_value() );
+    double cons_multiplier = 1.0 + ( cache.mastery() * mastery.divine_bulwark_2 -> effectN( 1 ).mastery_value() );
```

Comment:
You shouldn't need to check for cons_2 -> ok(), effectN() will silently return 0 if the spell isn't found.

(it's also a bit weird to check for cons_2 but not for divine_bulwark_2 since the latter requires a higher level than the former)